conjure/unionwriter.go

```diff
@@ -74,7 +76,7 @@ func writeUnionType(file *jen.Group, unionDef *types.UnionType, genAcceptFuncs b
 				jen.Nil(), snip.FmtErrorf().Call(jen.Lit("unknown type %s"), jen.Id(unionReceiverName).Dot("typ"))))
```

Comment:
If we quote the field name, then would likely want to do it for the "unknown type" output here as well